### PR TITLE
added previous speakers page

### PR DESCRIPTION
**Pull request by dstephens3**

### CODE REVIEW

**Review by harperiscoding — CHANGES_REQUESTED** [review]
Lets not add a :speaker param to the events and simply maintain a list of past events. I'm sure all the events will have the speakers either in the name or in the description. This should make your job a lot easier.

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -7,6 +7,10 @@ def index
     @events = Event.all.order(:date)
   end
 
+  def past_speakers
+    @events = Event.where("date < ?", Date.current).all
```
Comment (harperiscoding) [inline app/controllers/events_controller.rb:11]: is there a prettier way to write this? it's kinda obtuse rn.

File: app/views/events/_form.html.erb [hunk 2]
```diff
@@ -21,6 +21,11 @@
     <%= form.text_area :description, id: :event_description %>
   </div>
 
+  <div class="field">
+    <%= form.label :speaker %>
+    <%= form.text_area :speaker, id: :event_speaker %>
```
Comment (harperiscoding) [inline app/views/events/_form.html.erb:26]: text_area is used for large blocks of text, text_field would be better in cases like this.

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -19,6 +20,7 @@
       <tr>
         <td><%= event.name %></td>
         <td><%= event.description %></td>
+        <td><%= event.speaker %></td>
```
Comment (harperiscoding) [inline app/views/events/index.html.erb:23]: again, don't think we need to add a whole new field to events.

**Review by harperiscoding — CHANGES_REQUESTED** [review]

**Comment by dstephens3** [comment]
If there is no speaker param then how do we know there is a speaker's name in the description. We do not have to display the column either. This would also make it very hard to automatically maintain a list of past speakers. If you want to get rid of this param then we should just have a past events page instead of a past speakers page.

**Pull request closed without merging**